tests/generator/bug39614.cs

```diff
@@ -1,6 +1,8 @@
 using System;
 using Foundation;
 
+namespace FixWarningAboutNoNamespace {
```

Comment:
was not that test to check for a warning when no namespace is present ?
adding one seems to contradict that goal

Reply:
No, this test was to make sure the binding worked (https://bugzilla.xamarin.com/show_bug.cgi?id=39614) - https://github.com/xamarin/xamarin-macios/pull/3019/files#diff-acbf46b95c95f207677f4f5d23a59161L59

I added the namespace here because the NUnit tests are by default asserting that there are no warnings (Makefile-based tests didn't assert that), and this seemed like the easiest way to make that assert pass.

Reply:
```
commit cc88995a895daac053b0b03e590bae48de3c73a7
Author: Alex Soto <dalexsoto@gmail.com>
Date:   Wed Apr 13 08:53:38 2016 -0500

    [generator] Fix 39614 - Error generating class for generic type in iOS Binding project (#476)
    
    https://bugzilla.xamarin.com/show_bug.cgi?id=39614
    
    We generate invalid C# code if no namespace is provided in the
    ApiDefinition.cs file when generating generc ObjC types
    
    Added generator test and fix

```

Reply:
OK, I see, I got it backwards. I'll fix.

Reply:
Fixed: 0c435f6d53ccd5e84746d3e776d795cc55dd5e70